Clinical-Genomics/scout
# Sort ClinVar submission variants and case data by ascending "Last Evaluated" date

Fix #2519

Introduced changes:
- Expose link to Clinvar submissions in Institute page sidebar. In current master you need to expand sidebar "shared data" to see it.
- Order Submissions variants by ascending "last_evaluated" date.
- Order case data according to the order of variants 

**How to test**:
1. Pin and submit (with casedata) at least 2 variants without modifying the "last_evaluated" field in the submission form.
1. Note the order in which these variants are displayed in the submission object list, for instance this:
![image](https://user-images.githubusercontent.com/28093618/116210817-81f69380-a743-11eb-88a7-e3527ede045a.png)

1. In the database, edit the second variant with a date older than today ("clinvar" collection).
1. Make sure that both the second variant and its case data are shown first now in the submission object


**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
